### DISCUSSION TITLE
one wallet

### DISCUSSION THREAD

**Discussion by NicNem55** [Chia-Network/chia-blockchain#2094] [Q&A]
can I use one wallet on different machines in different networks?

**Comment by jjhesk** [comment]
yes, you should able to farm from different HDD and receive the rewards into one wallet.  The context is mentioned. The plot id is generated by hashing the plot public key and the pool public key. Creating a block with a proof of space requires signing with both the plot key and the pool key. Therefore the pool may not be changed after creating the plot.
